test/functional/test-3p-frame.js

```diff
@@ -155,11 +163,17 @@ describe('3p-frame', () => {
     expect(locationHref).to.not.be.empty;
     const docInfo = documentInfoForDoc(window.document);
     expect(docInfo.pageViewId).to.not.be.empty;
-    const amp3pSentinel = iframe.getAttribute('data-amp-3p-sentinel');
+    let amp3pSentinel;
+    if (iframeContextInName) {
```

Comment:
These tests need to be done with both `iframeContextInName` set to false and set to true. As it is, we'll never test the `true` case.